app-devel/mise/autobuild/defines

```diff
@@ -0,0 +1,7 @@
+PKGNAME=mise
+PKGDES="Polyglot runtime and development tool version manager"
```

Comment:
Runtime and development toolchain version manager for multiple programming languages